.github/workflows/publish-to-pypi.yml

```
@@ -0,0 +1,39 @@
+name: Publish Modin wheel to PyPI
+
+on:
+  schedule:
+    - cron: "42 0 * * WED"
+  push:
```

I think this got deleted somewhere. We need it so that this triggers for every new tag.
```suggestion
  push:
    tags:        
      - '*'
```